src/chainstate/burn/operations/transfer_stx.rs

```diff
@@ -224,7 +226,7 @@ impl TransferStxOp {
             warn!("Invalid TransferStxOp, sender is recipient");
             return Err(op_error::TransferStxSelfSend);
         }
-
+        increment_transfer_stx_op();
```

Comment:
This should be somewhere in the top-level `Burnchain::sync()` method, such as right after when a block gets processed.  There's no guarantee anywhere that this method will be called only once.

Reply:
Removed